doc/source/whatsnew/v0.21.0.txt

```diff
@@ -106,6 +106,8 @@ Reshaping
 
 Numeric
 ^^^^^^^
+- DataFrame.interpolate was not respecting limit_direction when using the default limit=None (unlimited). Specifically, it would always use limit_direction='forward' even when limit_direction was set otherwise. Now default limit=None will work with other directions. :issue:`16282`
```

Comment:
I think 0.20 was a typo. Probably mean 0.20.2. There's a `whatsnew/v0.20.2.txt`. You can move this section there (and make it a bit shorter. Typically just a sentence, and people can click on the link to the issue if they're interested in more).

And yeah, just a comment when you're ready for review, or when you notice that all the checkmarks at the bottom are green and it's ready for review.